tests/deployment/aws_lambda/test_aws_lambda_deployment_operator.py

```diff
@@ -96,27 +96,27 @@ def return_predict_func(_):
     with pytest.raises(RuntimeError):
         predict("Invalid Input Type", None)
 
-    assert predict({"headers": [], "body": 'test'}, None) == {
-        'body': 'test',
-        'statusCode': 200,
+    assert predict({"headers": [], "body": "test"}, None) == {
+        "body": "test",
+        "statusCode": 200,
     }
 
 
-@patch('shutil.rmtree', MagicMock())
-@patch('shutil.copytree', MagicMock())
-@patch('bentoml.yatai.deployment.aws_lambda.utils.cleanup_build_files', MagicMock())
-@patch('bentoml.yatai.deployment.aws_lambda.utils.call_sam_command', autospec=True)
+@patch("shutil.rmtree", MagicMock())
+@patch("shutil.copytree", MagicMock())
+@patch("bentoml.yatai.deployment.aws_lambda.utils.cleanup_build_files", MagicMock())
+@patch("bentoml.yatai.deployment.aws_utils.call_sam_command", autospec=True)
 def test_init_sam_project(mock_call_sam, tmpdir):
```

Comment:
@yubozhao I need your help here.I am not able mock `call_sam_command` here.
Test fails as init_sam_project calls real funciton and gets return value 1